dsl/src/commonTest/kotlin/com/freeletics/flowredux/dsl/StateMachine.kt

```diff
@@ -1,16 +1,20 @@
 package com.freeletics.flowredux.dsl
 
+import kotlin.coroutines.EmptyCoroutineContext
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.FlowPreview
 
 @OptIn(ExperimentalCoroutinesApi::class, FlowPreview::class)
 fun StateMachine(
         builderBlock: FlowReduxStoreBuilder<TestState, TestAction>.() -> Unit
 ): FlowReduxStateMachine<TestState, TestAction> {
-    val sm: FlowReduxStateMachine<TestState, TestAction> = object : FlowReduxStateMachine<TestState, TestAction>(
-            CommandLineLogger,
-            TestState.Initial){
-
+    val sm = object : FlowReduxStateMachine<TestState, TestAction>(
+        TestState.Initial,
+        CoroutineScope(Dispatchers.Unconfined),
```

Comment:
When I used `Dispatchers.Default` here some tests fail. I'm not sure if this is just because of how the tests are written or hints at some problem in the implementation of the DSL code.